Package/zip/Zip.Packaging.targets

```diff
@@ -9,8 +9,8 @@
     <Zip Files="@(ZipFiles)" ZipFileName="$(PackageDir)\IronPython.$(PackageVersion).zip" WorkingDirectory="$(StageDir)" />
 
     <ItemGroup>
-      <StdLibFiles Include="$(StageDir)\Lib\**\*.*" />
+      <StdLibFiles Include="$(StageDir)\$(StdLibDirName)\**\*.*" />
```

Comment:
Apart from the nuget package, all packages produced by `./make package` are platform specific. The zip package was the same regardless where it was produced, but it was only usable in Windows. This was something I wanted to open a separate issue about, but we can discuss it here. I was about to propose that builds on Linux and macOS produce zip packages that are usable on Linux and macOS, respecitvely (with the target platform as part of the name), and then become part of the artifact set of the official published milestones. Currently there is no way to get IronPython on .NET 6 installed on Linux or macOS (short from the `Install-IronPython.ps1` hack for developers). This will get better with dotnet tool support in place, but still, zipfiles have their niche.

By the way, I see no value in the StdLib zip archive. What is it for? Can we ditch it?

Reply:
> The zip package was the same regardless where it was produced, but it was only usable in Windows. This was something I wanted to open a separate issue about, but we can discuss it here. I was about to propose that builds on Linux and macOS produce zip packages that are usable on Linux and macOS, respecitvely (with the target platform as part of the name), and then become part of the artifact set of the official published milestones.

Aside from the the launcher executables (ipy.exe and ipy) what would be different in the linux/macos zip packages? I guess we would need `Mono.Unix.dll`, the `ipy.deps.json` and the `runtimes` folder (which could all be added if we opted for a single package).

> By the way, I see no value in the StdLib zip archive. What is it for? Can we ditch it?

I guess the main use case (in my mind) is a zip you can use for embedded scenarios:
```c#
engine.SetSearchPaths(new List<string>() { "IronPython.StdLib.3.4.0-beta1.zip" });
engine.Execute("import os");
```

Although I don't know if anyone is using it as such. In my case I just download the nuget StdLib and zip it up during my build process (would probably be nice to have an IronPython.StdLib.Zip nuget package).

Interestingly looking at the download stats for the beta1 release assets, the StdLib zip got downloaded more often than the zip? Go figure... (Not sure if you have access - https://api.github.com/repos/IronLanguages/ironpython3/releases)

Reply:
> Aside from the the launcher executables (ipy.exe and ipy) what would be different in the linux/macos zip packages? I guess we would need `Mono.Unix.dll`, the `ipy.deps.json` and the `runtimes` folder (which could all be added if we opted for a single package).

In case of a single zip package, this is exactly what would be needed, though the package would contain some files that are useless on any given platform (like `ipy.exe` on Linux/macOS, or the whole `runtimes` on Windows). In case you want to stick with a single, one-size-fits-all zip package, I think it would be better if you publish the one created on macOS or Linux, rather than on Windows: it seems that it would fix #1472. The zip command on macOS/Linux archives files with proper execution permissions.

But another thing: aren't `ipy` launchers on Linux/macOS different? Also: what about line endings in StdLib?

In case of separate zip files, `ipy.deps.json` seems not needed if `libMono.Unix.dylib` and `libMono.Unix.so` are put next to `Mono.Unix.dll`.

> I guess the main use case (in my mind) is a zip you can use for embedded scenarios:

OK, it makes sense. I haven't thought of it.

> Interestingly looking at the download stats for the beta1 release assets, the StdLib zip got downloaded more often than the zip?

Not an explanation for the phenomenon, but a piece from my experience: when I first downloaded the zips, I thought that the StdLib zip is the complete IronPython + StdLib, while the other one is just vanilla IronPython. It took a download to discover that I have downloaded the wrong zipfile.